README.md

```diff
@@ -223,13 +236,43 @@ Make sure that you replace "{repo-root-dir}" with the repository root directory
 `provider_authority: "0.0.0.0:4010"`<br>
 `invehicle_digital_twin_uri: "http://0.0.0.0:5010"`<br><br>
 1. In the top window, run:<br><br>
-`./in-vehicle-digital-twin`<br>
+`./invehicle-digital-twin`<br>
 1. In the middle window, run:<br><br>
 `./seat-massager-provider`<br>
 1. In the bottom window, run:<br><br>
 `./seat-massager-consumer`<br>
 1. Use control-c in each of the windows when you wish to stop the demo.
 
+### <a name="streaming-sample">Streaming Sample</a>
+
+The following instructions are for the demo for streaming.
+
+Steps:
+
+1. The best way to run the demo is by using three windows: one running the In-Vehicle Digital Twin, one running the Digital Twin Provider and one running the Digital Twin Consumer.
```

Comment:
since much of the setup is copied and pasted for all of these demos, can we move it out into a "common setup" section?

Reply:
There are suttle differences and I expect these to diverge as Ibeji becomes more mature.

Reply:
Leaving as is for now.